docs/Using-Fleet/CIS-Benchmarks.md

```diff
@@ -64,58 +37,87 @@ To apply the policies on a specific team use the `--policies-team` flag:
 fleetctl apply --policies-team "Workstations" -f cis-policy-queries.yml
 ```
 
-## CIS Apple macOS 13.0 Ventura Benchmark
+## Limitations
+Fleet's current set of benchmarks only implements benchmark *auditing* steps that can be *automated*.
 
-You can import macOS 13 Ventura CIS Benchmark the following way:
-```sh
-# Download policy queries from Fleet's repository (e.g. for macOS 13)
-wget https://raw.githubusercontent.com/fleetdm/fleet/main/ee/cis/macos-13/cis-policy-queries.yml
+In practice, Fleet is able to cover a large majority of benchmarks:
+* macOS 13 Ventura - 96 of 104
+* Windows 10 Enterprise - TODO
 
-# Apply the downloaded policies to Fleet.
-fleetctl apply -f cis-policy-queries.yml
-```
+For a list of specific checks which are not covered by Fleet, please visit the section devoted to each benchmark.
+
+### Audit vs. remediation
+Each benchmark has two elements:
+1. Audit - how to find out whether the host is in compliance with the benchmark
+2. Remediation - if the host is out of compliance with the benchmark, how to fix it
+
+Since Fleetd is currently read-only without the ability to execute actions on the host, Fleet does not implement the remediation portions of CIS benchmarks.
+
+To implement automated remediation, you can install a separate agent such as Munki, Chef, Puppet, etc. which has write functionality.
```

Comment:
Actually, the way to automatically remediate all of the benchmarks is via MDM (via deploying configuration profiles and scripts).

Reply:
Mhm... this is mentioned below:
```
Using MDM is the recommended way to manage and enforce CIS benchmarks. To learn how to set up MDM in Fleet, visit [here](/docs/using-fleet/mdm-setup).

```

Reply:
haha yeah now that we have an MDM, I want to recommend our approach which is better. (Fleet MDM + BYO script agent)